docs/src/whatsnew/3.1.rst

```diff
@@ -106,16 +112,32 @@ This document explains the changes made to Iris for this release
    :class:`~iris.coords.AuxCoord`, :class:`~iris.coords.CellMeasure`, and
    :class:`~iris.cube.Cube` instances. (:pull:`4227`)
 
+   .. _fast_nc_load:
+
 #. `@pp-mo`_ and `@lbdreyer`_ optimised loading netcdf files, resulting in a
-   speed up when loading with a single :func:`~iris.NameConstraint`. Note, this
-   optimisation only applies when matching on standard name, long name or
+   speed up when loading with a single :func:`~iris.NameConstraint`.
+
+   For example, ``cube = iris.load('x.nc', NameConstraint('air_temperature'))``.
+
+   Note that this  optimisation only applies when matching on standard name, long name or
    NetCDF variable name, not when matching on STASH.
    (:pull:`4176`)
 
 💣 Incompatible Changes
 =======================
 
-#. N/A
+.. _minimised_imports:
+
+#. The :ref:`adoption of 'isort' <isort_adopt>`  means that subpackage loading is everywhere minimised.
```

Comment:
`  means that subpackage loading is everywhere minimised.` -> ` has significantly reduced the import time of Iris subpackages.`